docs/building-modules/keeper.md

```diff
@@ -0,0 +1,89 @@
+# Keepers
+
+## Pre-requisite Reading
+
+- [Introduction to SDK Modules](./intro.md)
+
+## Synopsis
+
+`Keeper`s refer to a Cosmos SDK abstraction whose role is to manage access to the subset of the state defined by various modules. `Keeper`s are module-specific, i.e. the subset of state defined by a module can only be accessed by a `keeper` defined in said module. If a module needs to access the subset of state defined by another module, a reference to the second module's internal `keeper` needs to be passed to the first one. This is done in `app.go` during the instantiation of module keepers.
```

Comment:
Perhaps add a line describing what the underlying KVStore is? https://github.com/tendermint/iavl